nixos/modules/services/audio/mpd.nix

```diff
@@ -62,6 +60,14 @@ in {
         '';
       };
 
+      playlistDirectory = mkOption {
+        type = types.path;
```

Comment:
As is, there should be no difference as far as `types.path` is concerned. The check would accept both.